proto/ibc/core/commitment/v1/commitment.proto

```diff
@@ -26,8 +26,6 @@ message MerklePrefix {
 // arbitrary structured object (defined by a commitment type).
 // MerklePath is represented from root-to-leaf
 message MerklePath {
-  option (gogoproto.goproto_stringer) = false;
```

Comment:
I removed this so that the proto compiler generated an implementation of `String` because that is needed when we do [this](https://github.com/cosmos/ibc-go/blob/d0f7773317daa4c242e10b7a4efe915fbf9c7e68/testing/solomachine.go#L206).

Reply:
I think the linked test should be updated to use a valid path for misbehaviour. A marshaled merkle path is not an expected singing path for solomachines

Reply:
It turns out that we might need to remove this because of [this](https://github.com/cosmos/ibc-go/blob/c849a78e0df9b8cd4a658f60d5bf171161bc28fd/modules/core/23-commitment/types/codec.go#L36-L39) and [this](https://github.com/cosmos/ibc-go/blob/c849a78e0df9b8cd4a658f60d5bf171161bc28fd/modules/light-clients/06-solomachine/misbehaviour_handle.go#L44-L46) (why would we require the path to be proto-marshalled when submitting misbehaviour?).

Reply:
Allowing it to be proto marshaled makes sense to me, but solo machine should not be proto marshaling the path when submitting misbeahviour, I think that might be a bug